pkg/spi-shared/config/config.go

```diff
@@ -104,11 +115,21 @@ func (c PersistedConfiguration) inflate() (Configuration, error) {
 		ttlStr = "1h"
 	}
 
+	if c.VaultHost == "" {
+		conf.VaultHost = DefaultVaultHost
+	} else {
+		conf.VaultHost = c.VaultHost
+	}
+
 	ttl, err := time.ParseDuration(ttlStr)
 	if err != nil {
 		return conf, err
+	}
 
+	if saTokenPath, ok := os.LookupEnv("SA_TOKEN_PATH"); ok {
```

Comment:
This env var is not documented in the description of that config value. I think it would be worth mentioned there.

Reply:
done